astropy/io/fits/tests/test_fitsdiff.py

```diff
@@ -0,0 +1,174 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import sys
+
+import numpy as np
+
+from . import FitsTestCase
+from ..hdu import PrimaryHDU
+from ..scripts import fitsdiff
+from ....tests.helper import pytest
```

Comment:
Note to @astrofrog , you'll need to change this too for #5694 if this gets merged first.